Geospatial.md

```diff
@@ -94,6 +94,39 @@ Bounding box is defined as the thrift struct below in the representation of
 min/max value pair of coordinates from each axis. Note that X and Y Values are
 always present. Z and M are omitted for 2D geospatial instances.
 
+Writers should follow the guidelines below when calculating bounding boxes in
+the presence of edge cases.
+
+* `null` instance: Skip it and continue processing the remaining 
+  geospatial instances. Do not produce a bounding box if all instances are null.
+* Non-`null` instance with [invalid geospatial values](#invalid-geospatial-values):
+  * X and Y: Skip any invalid X or Y value and continue processing the 
+    remaining X or Y values. Do not produce a bounding box if all X or all Y 
+    values are invalid.
+
+  * Z: Skip any invalid Z value and continue processing the remaining Z values.
+    Omit Z from the bounding box if all Z values are invalid.
+
+  * M: Skip any invalid M value and continue processing the remaining M values.
+    Omit M from the bounding box if all M values are invalid.
+
+Readers should follow the guidelines below when examining bounding boxes. 
+Parquet does not permit `null` or `NaN` values in bounding boxes, whether at 
+the overall bounding box level or within individual coordinate fields.
+
+* No bounding box: No assumptions can be made about the presence or validity 
+  of coordinate values. Readers may need to load all individual coordinate 
+  values for validation.
+
+* A bounding box is present:
+    * X and Y: Both X and Y of the bounding box must be present.
```

Comment:
```suggestion
    * X and Y: Both X and Y of the bounding box must be present. If any X or Y
      value is invalid, this bounding box is not reliable and cannot be used.
```

Reply:
I’m hesitant to merge this because I want to avoid mentioning invalid values in the bounding box; otherwise, we would need to create a separate section to define them.

Reply:
The guidelines here is for readers to determine whether a bbox is reliable. So I think `value is present` is not sufficient.